app/models/work.rb

```diff
@@ -0,0 +1,38 @@
+class Work < ApplicationRecord
+  def self.spotlight
+    return Work.all.sort_by { |media| media.votes.count }.reverse.first
+  end
+
+  def self.albums
+    return Work.where(category: "album").sort_by { |album| album.votes.count }.reverse
+  end
+
+  def self.top_ten_albums
+    return Work.albums.select { |album| album.votes.count > 0 }
+  end
+
+  def self.books
+    return Work.where(category: "book").sort_by { |movie| movie.votes.count }.reverse
+  end
+
+  def self.top_ten_books
+    return Work.books.select { |book| book.votes.count > 0 }
+  end
+
+  def self.movies
+    return Work.where(category: "movie").sort_by { |book| book.votes.count }.reverse
```

Comment:
Just noting these solutions are better using the database functions like `.order` or `.where` etc.  It's much more efficient to let Postgres do the sorting and filtering for you before Rails gets the list.  This works at present, but won't scale well.